test/constants.json

```diff
@@ -30,9 +29,9 @@
 		"contractsPath": "/contracts/ExampleContract.aes",
 		"nodeModules": "/node_modules",
 		"dockerEntryPoint": "/docker/entrypoint.sh",
-		"dockerEpochNode1": "/docker/epoch_node1_mean15.yaml",
-		"dockerEpochNode2": "/docker/epoch_node2_mean15.yaml",
-		"dockerEpochNode3": "/docker/epoch_node3_mean15.yaml",
+		"dockernodeNode1": "/docker/epoch_node1_mean15.yaml",
```

Comment:
dockernodeNode1 sounds odd to me, can we rename it to only localNode or dockerNode? If we rename it here, we should change it everywhere it is used